translations/units.tex/pl.po

```diff
@@ -124,6 +124,21 @@ msgid ""
 "\\textbf{Reinforcing} requires that your Town has a Citadel in addition to a Dwelling of that Unit's Level.\\par\n"
 "\n"
 msgstr ""
+"\\hypertarget{Units}{In}\\index{Units} Oprócz Bohatera i powiązanej z nim talii, gracze posiadają także \\textbf{talię Kart Jednostek}, która reprezentuje armię poruszającą się z ich Bohaterami.\n"
+"Każda Frakcja ma dostęp do \\textbf{7 różnych Jednostek} o unikalnych zdolnościach i atrybutach.\n"
+"Jednostki są niezbędne do wygrywania walk i realizacji celów scenariusza.\n"
+"Każdy scenariusz posiada instrukcje wskazującą, które Karty Jednostek powinny być zawarte w twojej początkowej Talii Jednostek.\n"
+"Talia ta powinna być oddzielona od reszty Jednostek twojej Frakcji.\\par\n"
+"Każda Jednostka Frakcji jest dwustronna, posiada słabszą \"Garstka\" stronę, oraz silniejszą \"Grupa\" stronę.\n"
```

Comment:
sugestia żeby zamienić miejscami, a cudzysłowy powinny ,,wyglądać tak'', żeby to zostało przemielone na polskie cudzysłowy
 
```suggestion
"Każda Jednostka Frakcji jest dwustronna, posiada słabszą stronę ,,Garstka'', oraz silniejszą stronę ,,Grupa''.\n"
```

Reply:
Będe wiedział na przyszłość o cudzysłowach, zamiana zasadna